src/components/container/PartnerBanner/PartnerBanner.scss

```diff
@@ -0,0 +1,100 @@
+.PartnerBanner {
+    width: 100%;
+    overflow: hidden;
+    background-color: var(--color-bunting); 
+
+    div {
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        // transform: translateX(-10vw);
+        animation: slide 20s 1s linear infinite;
+

```

Comment:
don't we want to add a gap property here to make the prgh logos not so close to each other? and also it prevents from having an empty space at the end of moving, - at least during my tests))

Reply:
I added margin for the image  :)